tap_nice_incontact/schemas/qm_questions.json

```diff
@@ -0,0 +1,112 @@
+{
+  "type": ["null", "object"],
+  "additionalProperties": false,
+  "properties": {
+    "actualScore": {
+      "type": ["null", "number"]
+    },
+    "answer": {
+      "type": ["null", "string"]
+    },
+    "questionId": {
+      "type": ["null", "string"]
+    },
```

Comment:
Additionally, can we remove nulls from some of these fields? An API for questions should at least have this field filled in all them time.

Reply:
yeah the problem is that there API documentation doesn't provide this information, so I decided to be less strict in the schema json and accept nulls everywhere. what do you suggest i do, try to guess which fields are non-nullable the best i can?

Reply:
Yea good question. I think we can infer at least 1 unique field (like questionId). 

We also can also just remove nulls from the entire schema and run it with target-postgres. It'll complain the column is non-nullable but encountered a null.

Reply:
ahh okay cool, yeah thats what i did in the latest comment (added non-null to both of these fields)

```"workflowinstanceId": {
      "type": ["string"]
    },
    "questionId": {
      "type": ["string"]
    },```